packages/thirdweb/src/wallets/in-app/core/wallet/enclave-wallet.ts

```diff
@@ -167,11 +167,11 @@ export class EnclaveWallet implements IWebWallet {
         transaction.maxPriorityFeePerGas =
           typeof tx.maxPriorityFeePerGas === "bigint"
             ? toHex(tx.maxPriorityFeePerGas)
-            : undefined;
+            : toHex("0");
         transaction.type = 2;
       } else {
         transaction.gasPrice =
-          typeof tx.gasPrice === "bigint" ? toHex(tx.gasPrice) : undefined;
+          typeof tx.gasPrice === "bigint" ? toHex(tx.gasPrice) : toHex("0");
```

Comment:
this is dangerous, a lot of tools like at the presence of gasPrice to know if its a legacy tx

we cant risk sending all values, has to be one or the other